packages/testing/src/MockRouter.tsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react'
-// We're bypassing the `main` field in `package.json` because we're
-// replacing imports of `@redwoodjs/router` with this file, and not doing so
-// would cause an infinite loop.
+// Bypass the `main` field in `package.json` because we alias `@redwoodjs/router`
+// for jest and Storybook. Not doing so would cause an infinite loop.
 // See: ./packages/core/config/jest.config.web.js
+// @ts-ignore
+import { Private, Route } from '@redwoodjs/router/dist/index'
 export * from '@redwoodjs/router/dist/index'
 
 export const routes: { [routeName: string]: () => string } = {}
 
+const getPrivateRoutes = (children: React.ReactNode) =>
```

Comment:
Do you expect any other nested type of routes in the Router in the (near) future? If so it could be worthwhile to make the function recursively look for routes?

If not, ignore me!

Reply:
I don't think so, we really want to keep it flat, but this might be a great place to adopt the `@redwoodjs/structure`'s implementation instead of duplicating it across Routers.